apps/base-docs/docusaurus.config.js

```diff
@@ -1,3 +1,7 @@
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
I think we don't need to call `.config` twice, just the second invocation should be enough